docs/using-the-compiler.rst

```diff
@@ -439,6 +439,8 @@ Input Description
           "extCalls": "trusted",
           // Choose which types of invariants should be reported to the user: contract, reentrancy.
           "invariants": ["contract", "reentrancy"],
+          // Choose whether to output all proved targets. The default is `false`.
+          "showProved": true,
```

Comment:
`showProved` or `showProvedSafe`? Are they the same thing?

Reply:
yea that's the same thing